lib/node_modules/@stdlib/math/base/special/fast/alpha-max-plus-beta-min/README.md

```diff
@@ -133,6 +133,85 @@ for ( i = 0; i < 100; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/math/base/special/fast/alpha-max-plus-beta-min.h"
+```
+
+#### stdlib_base_fast_ampbm( x, y ) 
+
+Computes the hypotenuse using the alpha max plus beta min algorithm.
+
+```c
+double h = stdlib_base_fast_ampbm( -5.0, 12.0 ); // returns ~13.5
```

Comment:
```suggestion
double h = stdlib_base_fast_ampbm( -5.0, 12.0 );
// returns ~13.5
```

Return annotations always go on the next line.

Reply:
added return annotation to new line